Java/cp/Intersection_of_Two_Arrays.java

```diff
@@ -0,0 +1,101 @@
+/*
+  - Intersection :
+       We have to find intersection of two arrays :
+       The intersection of two arrays is a list of numbers which are present in both the arrays
+     Example :
+       input : Enter the number of elements you want in the first array : 4
+	           Enter the elements in the 1st array : 1 4 3 2
+	 		   Enter the number of elements you want in the second array : 3
+	 		   Enter the elements in the 2nd array : 1 2 3
+      output : The intersection of two arrays is : 1 2 3
+               Intersection of both arrays is [1 2 3] as these elements are present in both the arrays
```

Comment:
Remove this and,
- Add Multi-Line Comments 
- Add a beginner-friendly description at the top